optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/persistence/MeetingSchedulingXlsxFileIO.java

```diff
@@ -486,6 +542,15 @@ private void writeMeetings() {
                         meeting.getPreferredAttendanceList().stream()
                                 .map(preferredAttendance -> preferredAttendance.getPerson().getFullName())
                                 .collect(joining(", ")));
+                List<MeetingAssignment> meetingAssignmentList = meetingAssignmentMap.get(meeting);
+                if (meetingAssignmentList.size() != 1) {
+                    throw new IllegalStateException("Impossible state");
```

Comment:
another soft suggestions: error message could be more descriptive about what impossible state happened.

Reply:
Agreed, will fix in Parameterization branch